### DISCUSSION TITLE
Aspx file downloading instead of displaying on web page

### DISCUSSION THREAD

**Discussion by WinterRice** [community/community#22239] [Other Feature Feedback, Questions, & Ideas]
<p>Hello!  I just uploaded my first Visual Studio 2019 project to Github.  After successfully uploading it, I built the following Github Pages website:</p>
<p><a href="https://winterrice.github.io/Login_VS/" rel="nofollow noopener">https://winterrice.github.io/Login_VS/</a>LogIn2/</p>
<p>(NOTE that I am calling it from the subfolder “LogIn2”.)</p>
<p>BE AWARE that if you load this webpage it will automatically erroneously save the file “LogIn.aspx” to your download folder, so just please be aware of that in advance.</p>
<p>So, that is the issue, that when I load the webpage it dowloads the LogIn.aspx file that the index.html file calls instead of displaying it in the browser.</p>
<p>I tried uploading to Netlify as well and the same thing occurs.</p>
<p>ANY help would be greatly appreciated!  Thank you in advance for any advice.</p>

**Comment by WinterRice** [comment]
<p>Oh, and I forgot to add that when I run the webpage locally (in Visual Studio 2019) on my PC it displays everything perfectly, instead of downloading the Login2.aspx file.</p>

**Comment by ChrisHammond** [comment]
<p>Did you ever figure this out? I’m experiencing this</p>

**Comment by WinterRice** [comment, marked as answer]
<p>Hey, Chris!  The only solution was to find a hosting site that supports <a href="http://ASP.NET" rel="noopener nofollow ugc">ASP.NET</a>.  And I was looking for a free site.  So I went with <a href="http://somee.com" rel="noopener nofollow ugc">somee.com</a> which has some advertising at the bottom of the site you create using it.</p>
<p>I have not looked at this program or done any programming in months by the way.</p>

**Comment by otter-computer** [comment]
<p>This is the answer I’m afraid. GitHub Pages only supports static sites—sites made with pure HTML, CSS, and JS. If your site uses a server-side language like Python, PHP, or .NET you’ll need to find an alternate hosting provider.</p>

**Comment by ChrisHammond** [comment]
I would be completely fine if GitHub pages simply served the ASPX file as HTML. I only have them in place because of a redirect need for old URLs
